DESCRIPTION

```diff
@@ -18,7 +18,9 @@ Authors@R:
             role = c("aut")),
       person(given = "Jorge",
             family = "Cimentada",
-            role = c("aut"))                      
+            role = c("aut")),
+      person(given = "Sachet", family = "Antoine",
+             email = "antoine.sac@gmail.com", role = "ctb")                         
```

Comment:
We so appreciate community contributions @antoine-sachet! 🙌 In this particular repo, we have not typically added folks to `DESCRIPTION` for PRs. This is definitely something that varies from package to package, but here, for context, there have been a number of other PRs of about this size that have not resulted in getting added as an official package contributor. The folks you see here listed are current and previous _maintainers_ of the package.

Reply:
No worries. I did see the other people were _maintainers_ (i.e."aut" and "cre"), but the "ctb" role is useful for smaller contributions not worthy of citation. Its use definitely varies from package to package :)